scss/elements/tables.scss

```
@@ -0,0 +1,23 @@
+.table {
+  background-color: #fff;
```

```suggestion
  table-layout: fixed;
  background-color: #fff;
```
This code is adjustment of table width. ✌️